content/articles/2019/09/15/lakka-2.3-with-retroarch-1.7.8.md

```diff
@@ -0,0 +1,213 @@
+
+Please note that the old cores are still present in the system, but should not be used anymore. They will be removed in a future Lakka release.
+
+## Flycast
+
+Flycast is a new and improved version of the Reicast Dreamcast emulator. Compatible with WinCE games with a performance increase up to 30% on both regular and WinCE titles, this libretro-exclusive core is the future of Dreamcast emulation.
+
+As a result, Reicast has been replaced by Flycast in Lakka 2.3.
+
+## Bsnes HD
+
+Bsnes HD is an updated and significantly faster variant of the regular bsnes core you already know. The major feature of this new core is [HD Mode 7](https://www.youtube.com/watch?v=xcirI5fySok), allowing you to increase the resolution of "Mode 7" textures in various SNES games.
+
+Please note that the name "Bsnes HD" is temporary and will likely change in the future.
+
+## Final Burn Neo
+
+In a similar way to Flycast and Reciast, Final Burn Neo is the new and improved version of the Final Burn Alpha core, which has been removed from Lakka 2.3 (except on Raspberry Pi 0 and 1).
+
+[Here](https://github.com/libretro/FBNeo/tree/a28d26f0418c7eb9771c1f026cfe1a6c141d2eb3/dats) are the DAT files used in the Final Burn Neo version used in Lakka 2.3.
+
+# Full changelog
+
+![Changelog](media/changelog.png)
+
+As usual all cores, databases, controller mappings and assets have been updated.
+
+- RetroArch 1.7.8 (full RetroArch changelog can be found [here](https://retroarch.com/?page=changelog))
+  - Playlist-based thumbnail downloader, on-demand thumbnail downloads
+  - AI Project (Real-time games translation)
+  - Disc Project
+  - XMB Improvements
+  - Improved onscreen notifications
+  - Shader Usability Changes
+  - Added an easter egg
+- New platforms and accessories
+  - New platform: Raspberry Pi 4
+  - New Raspberry Pi accessory: GPICase
+  - New Raspberry Pi accessory: Game HAT
+    - The Game HAT driver can be enabled manually from any existing Raspberry Pi Lakka installation by following the instructions [here](https://github.com/libretro/Lakka-LibreELEC/blob/master/packages/linux-drivers/game-hat/README.md)
+  - New platform: ROCKPro64
+  - New plaform: Vulkan-only Nvidia computers
+    - This variant of the Generic platform only supports Vulkan on computers with Nvidia-powered GPUs. While it runs some cores better (such as Dolphin), it also has a lot of incompatibles, removed cores
+    - You can find a link to the Vulkan-only Nvidia image on the 64-bit Generic download page of the website
+    - Thanks to Albert Malina for the port!
+- New cores (please note that not all cores are available on all platforms)
+  - Play!
+    - This new PS2 emulator is very experimental and doesn't run all games, don't expect it to run properly on your Lakka box yet
```

Comment:
Maybe add a link to [playable games](https://github.com/jpd002/Play-Compatibility/issues?q=is%3Aissue+is%3Aopen+label%3Astate-playable) ?